src/Core/src/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -140,5 +126,46 @@ public void Configure(HostBuilderContext context, IServiceProvider services)
 			{
 			}
 		}
+
+		class HotReloadBuilder : IMauiServiceBuilder
+		{
+			public string? IdeIp { get; set; }
+
+			public int IdePort { get; set; } = 9988;
+
+			public void ConfigureServices(HostBuilderContext context, IServiceCollection services)
+			{
+			}
```

Comment:
Empty because we are not actually registering anything with the services system.